deploy/clowdapp.yaml

```diff
@@ -342,6 +344,10 @@ parameters:
   name: RBAC_SCHEME
   required: true
   value: http
+- description: Skip the RBAC service entirely. All the identified requests will be assumed to be valid.
+  displayName: Bypass RBAC option enabled
+  name: BYPASS_RBAC
+  value: false
```

Comment:
might need to make this the string `"false"` just in case of yaml wonkiness.

Reply:
Whoops, yeah. I saw that in the PR check but I forgot about it :hand_over_mouth: